python/cuml/benchmark/bench_helper_funcs.py

```diff
@@ -104,5 +112,8 @@ def _build_treelite_classifier(m, data, arg={}):
 
 
 def _treelite_fil_accuracy_score(y_true, y_pred):
-    y_pred_binary = y_pred > 0.5
-    return cuml.metrics.accuracy_score(y_true, y_pred_binary)
+    if isinstance(y_pred, np.ndarray):
+        y_pred_binary = y_pred > 0.5
```

Comment:
Why is thresholding applied only to `np.ndarray`?

Reply:
The reason is this accuracy function is also used in `treelite` runs. If the result is from treelite, the y_pred is actually a float array